lightning/models/cookie.py

```diff
@@ -0,0 +1,38 @@
+from lightning_adapter.models import Cookie as Cookie_gen
+from ..exceptions import CookieException
+
+
+class Cookie(dict):
```

Comment:
This class should live under `lightning` package to prevent adding more imports in tests.

Reply:
fixed in 8c543aa7094c15a53e877f4037bc152f2c1099b3